scripts/img2img.py

```diff
@@ -249,8 +258,8 @@ def main():
                 for n in trange(opt.n_iter, desc="Sampling"):
                     for prompts in tqdm(data, desc="data"):
                         uc = None
-                        if opt.scale != 1.0:
-                            uc = model.get_learned_conditioning(batch_size * [""])
+                        if negative_prompt:
+                            uc = model.get_learned_conditioning(len(prompts) * [negative_prompt])
```

Comment:
You should handle the 'else' case as well. Just put the old uc = line back there.

Reply:
Why len(prompts) here instead of batch_size like before?

Reply:
> You should handle the 'else' case as well. Just put the old uc = line back there.

That's right. Thank you for pointing that out.

> Why len(prompts) here instead of batch_size like before?

In my local environment (for mac), the default value of batch_size was 3. So I tried to get it from the len of prompts and did this.
Maybe that's not necessary in main branches.

(I am not good at English, so some parts may be wrong. Sorry.)

Reply:
Thanks! - LGTM.